pkgs/servers/web-apps/shiori/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Comment:
See #69229. I think `buildGoPackage` is preferred, though I also know little about go.

Reply:
I have tried with `buildGoPackage`, but it failed after compilation and tests succeeded :-/

Reply:
I have never used either so cannot really help. I just remembered seeing the pr I linked earlier

Reply:
it seems lots of other prs are using buildGoModule, so I wouldn't worry about it unless someone else suggests changing it

Reply:
Well, the Nixpkgs documentation does say that:

> The function `buildGoPackage` builds legacy Go programs, not supporting Go modules.

This gives the impression that `buildGoModule` is preferred ^^

Reply:
sure does